__tests__/costumes.test.js

```diff
@@ -8,6 +8,18 @@ const {
     getPool
 } = require(".././index");
 
+const { 
+    matchesDatabase,
```

Comment:
Not: as a general rule, in test utils, I recommend using functions that return the test data, that way they can be reused across multiple test files without reusing the same object. It's best if each test uses fresh data to ensure it's not possible to modify the same object in multiple tests

Reply:
So, import functions like this instead?: 
```
const getBallroomGown = () => {
    const ballroomGown = {
        name: "ballroom gown",
        category: "adult",
        gender: "female",
        size: "L",
        type: "dress",
        stockCount: 1,
        price: 150.99
    };
    return ballroomGown;
}

Reply:
Ok, I will do that.